areaOfObject.js

```diff
@@ -18,7 +18,7 @@ function areaOfShapes(Tshapes) {
 
 
     } else {
-        return 'shape doesnt exist';
+        return "shape doesn't exist'";
```

Comment:
areaOfShapes should take one shape at a time, but your array of objects will not give you the circle object by doing shapes['circle']

Reply:
you don't have to define the shapes in arrays, they could be individually defined.